ivy/functional/frontends/tensorflow/nn.py

```diff
@@ -123,3 +123,7 @@ def batch_normalization(x, mean, variance, offset, scale, variance_epsilon, name
     return x * ivy.astype(inv, x.dtype, copy=False) + ivy.astype(
         offset - mean * inv if offset is not None else -mean * inv, x.dtype
     )
+
+
+def conv_transpose(input, filters, output_shape, strides, padding="SAME", data_format="NCDHW", dilations=None, name=None):
+    return ivy.conv_transpose( input, filters, output_shape, strides, padding, data_format=data_format, dilations=dilations)
```

Comment:
I don't think there's a `conv_transpose` function in Ivy's functional API, so either you'll have to implement the `tensorflow` `conv_transpose` frontend as a composition of existing Ivy functions or extend Ivy's functional API by adding `ivy.conv_transpose` in a separate PR referring to the [docs](https://lets-unify.ai/ivy/contributing/4_open_tasks.html#ivy-api-extensions)